pipenv/utils.py

```diff
@@ -45,7 +45,7 @@
 
 # List of version control systems we support.
 VCS_LIST = ('git', 'svn', 'hg', 'bzr')
-SCHEME_LIST = ('http://', 'https://', 'ftp://', 'file://')
```

Comment:
Why is this getting added to the scheme list?

Reply:
erm, it shouldn't be necessary, sorry for the sloppy PR that was my early hope that just adding it to the scheme list would pick it up in the logic